### PR TITLE
Add Tammy to people

### PR DESCRIPTION
**Pull request by aneedalie**
:open_hands: :cake:

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -206,6 +206,9 @@ a:hover {
 .person ul li a {
   text-decoration:none;
 }
+.info {
+  margin: 15px 10px;
```
Comment (davepagurek) [inline style.css]: Do we want the slight offset?

Comment (aneedalie) [inline style.css]: Super small, but it removes the weird illusion that the text is coming before the picture

Comment (icechen1) [inline style.css]: Screenshot of how it looks like? :)

Comment (aneedalie) [inline style.css]: http://puu.sh/nKyUa/c794e59727.png :smile:

Comment (icechen1) [inline style.css]: Looks great to me :+1:

**Pull request merged**